packages/run-protocol/test/psm/test-psm.js

```diff
@@ -188,3 +207,51 @@ test('limit', async t => {
   // TODO Offer result should be an error
   // t.throwsAsync(() => await E(seat1).getOfferResult());
 });
+
+test('anchor is 2x stable', async t => {
+  const {
+    zoe,
+    feeMintAccess,
+    terms,
+    installs: { psmInstall },
+    runKit: { runIssuer, runBrand },
+    anchorKit: { brand: anchorBrand, issuer: anchorIssuer, mint: anchorMint },
+  } = t.context;
+  const anchorPerStable = makeRatio(200n, anchorBrand, 100n, runBrand);
+  const { publicFacet } = await E(zoe).startInstance(
+    psmInstall,
+    harden({ AUSD: anchorIssuer }),
+    { ...terms, anchorPerStable },
+    harden({ feeMintAccess }),
+  );
+  const giveAnchor = AmountMath.make(anchorBrand, 400n * 1_000_000n);
+  const seat1 = await E(zoe).offer(
+    E(publicFacet).makeSwapInvitation(),
+    harden({ give: { In: giveAnchor } }),
+    harden({ In: anchorMint.mintPayment(giveAnchor) }),
+  );
+  const runPayout = await E(seat1).getPayout('Out');
+  const expectedRun = minusAnchorFee(giveAnchor, anchorPerStable);
+  const actualRun = await E(runIssuer).getAmountOf(runPayout);
+  t.deepEqual(actualRun, expectedRun);
+  const liq = await E(publicFacet).getCurrentLiquidity();
+  t.true(AmountMath.isEqual(liq, giveAnchor));
+  const giveRun = AmountMath.make(runBrand, 100n * 1_000_000n);
+  trace('get stable ratio', { giveRun, expectedRun, actualRun, liq });
```

Comment:
I don't think it makes sense to leave `trace` statements in tests. If they're important enough to leave around, they should be converted to assertions.

Reply:
They should be using the logging that only turns on if the test fails. They aren't asserts. They are informational.

Reply:
`makeTracer` now supports disabling, so I did that.